jdat_notebooks/MRS_Mstar_analysis/JWST_Mstar_dataAnalysis_usecase.ipynb

```diff
@@ -9,46 +9,31 @@
+    "\n",
+    "**Note**: Ultimately, this notebook will include MIRI simulated data cubes obtained using MIRISim (https://wiki.miricle.org//bin/view/Public/MIRISim_Public)\n",
     "and run through the JWST pipeline (https://jwst-pipeline.readthedocs.io/en/latest/) of\n",
     "point sources with spectra representative of late M type stars.\n",
     "\n",
-    "For this example, we use a KMOS datacube of point sources in the LMC from Jones et al. (in prep)\n",
-    "to test the first part of the notebook, whilst MIRISim & the JWST pipeline is undergoing development.\n",
-    "\n",
-    "For the spectral analysis section: one star is represented by a dusty SED corresponding to the ISO SWS spectrum of\n",
-    "W Per from Kraemer et al. (2002) and Sloan et al. (2003) to cover the MRS spectral range 5-28 microns.\n",
-    "\n",
-    "Analysis of JWST spectral cubes requires extracting spatial-spectral features of interest and measuring their attributes. \n",
+    "## Introduction\n",
     "\n",
-    "**In this notebook we**:\n",
+    "This notebook analyzes one star represented by a dusty SED corresponding to the ISO SWS spectrum of\n",
+    "W Per from Kraemer et al. (2002) and Sloan et al. (2003) to cover the MRS spectral range 5-28 microns.  Analysis of JWST spectral cubes requires extracting spatial-spectral features of interest and measuring their attributes. \n",
     "\n",
-    "1) Process the datacube and automatically detect and extract spectra (summed over its spatial region) for all point sources in the cube.\n",
-    "- Read in a datacube generated at Stage 3 of the JWST pipeline or use near-IR data from KMOS as a representative example of an IR data cube.\n",
-    "- Use `photutils` to automatically detect sources in the continuum image.\n",
-    "- Use an aperture mask generated with `spectral-cube` to extract the spectra of each point source in the data cube.\n",
+    "The first part of the notebook will process the datacube and automatically detect and extract spectra (summed over its spatial region) for all point sources in the cube.  Then it will read in a datacube generated at Stage 3 of the JWST pipeline or use near-IR data from KMOS as a representative example of an IR data cube.  The analysis will use `photutils` to automatically detect sources in the continuum image and use an aperture mask generated with `spectral-cube` to extract the spectra of each point source in the data cube.\n",
     "\n",
-    "2) Data analysis using `specutils`\n",
-    "- Fit a model photosphere/blackbody to the spectra. \n",
-    "- Calculate the centroids, line integrated flux and equivalent width for each dust and molecular feature. \n",
+    "The second part of the notebook will perform data analysis using `specutils`.  Specifically, it will fit a model photosphere/blackbody to the spectra.  Then calculate the centroids, line integrated flux and equivalent width for each dust and molecular feature. \n",
```

Comment:
```suggestion
    "The second part of the notebook will perform data analysis using `specutils`.  Specifically, it will fit a model photosphere/blackbody to the spectra.  Then it will calculate the centroids, line integrated flux and equivalent width for each dust and molecular feature. \n",
```